tensorboard/webapp/widgets/dropdown/dropdown_test.ts

```diff
@@ -0,0 +1,148 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component, Input} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {MatSelectModule} from '@angular/material/select';
+import {By} from '@angular/platform-browser';
+import {DropdownComponent, DropdownOption} from './dropdown_component';
+
+/**
+ * Test class that uses the <tb-dropdown> component.
+ */
+@Component({
+  selector: 'testing-component',
+  template: `
+    <tb-dropdown
+      [value]="expId"
+      [options]="configOptions"
+      (selectionChange)="expChanged($event)"
+    ></tb-dropdown>
+  `,
+})
+class TestableComponent {
+  @Input() expId!: string;
+  @Input() configOptions!: DropdownOption[];
+  @Input() expChanged!: (event: {value: string}) => void;
```

Comment:
Also it's not obvious is test cases but I would also suggest rename `selectionChange` or `onSelectionChange` to be meaningful.

Reply:
Done. Thanks for the readability suggestion!